flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -2268,6 +2268,11 @@ void OmpAttributeVisitor::CreateImplicitSymbols(
 void OmpAttributeVisitor::Post(const parser::Name &name) {
   auto *symbol{name.symbol};
 
+  // if -fopenmp-default-none was given on the command line, act as if
+  // DEFAULT(NONE) was present at the directive.
+  bool HaveOpenMPDefaultNone = context_.languageFeatures().IsEnabled(
```

Comment:
This should be `haveOpenMPDefaultNone` (lowercase H at the start) to be consistent with the other local variables.

Reply:
Done!